src/components/admin/HearingToolbar.js

```diff
@@ -38,7 +36,7 @@ class HearingToolbar extends React.Component {
     ];
     const open = isOpen(hearing);
     if (open && hearing.published) {
-      statusLabel = <FormattedMessage id="published"/>;
+      statusLabel = (<Alert bsStyle="success"><Icon name="eye"/> <FormattedMessage id="published"/></Alert>);
```

Comment:
Ok. Should also file an issue about this in hel-bootstrap-3. Must use white text with success, like we use in danger.